src/wp-includes/blocks.php

```diff
@@ -1002,6 +1002,115 @@ function set_ignored_hooked_blocks_metadata( &$parsed_anchor_block, $relative_po
 	return '';
 }
 
+/**
+ * Runs the hooked blocks algorithm on the given content.
+ *
+ * This function is meant for internal use only.
+ *
+ * @since 6.6.0
+ * @access private
+ *
+ * @param string $content Serialized content.
+ * @param WP_Block_Template|WP_Post|array $context       A block template, template part, `wp_navigation` post object,
+ *                                                       or pattern that the blocks belong to.
+ * @return string The serialized markup with the ignored hooked blocks metadata applied.
+ */
+function apply_hooked_blocks_to_content( $content, $context, $callback = 'insert_hooked_blocks' ) {
```

Comment:
Open to renaming this function.

Reply:
How about `insert_hooked_blocks_into_content`? Or `inject_`?

We already have `insert_hooked_blocks` (which is the callback used by `make_*_block_visitor`). So maybe `insert_` for consistency's sake 🤔